target/linux/ath79/image/generic.mk

```diff
@@ -1475,6 +1475,110 @@ define Device/qihoo_c301
 endef
 TARGET_DEVICES += qihoo_c301
 
+define Device/qxwlan_e1700ac-v2-16m
+  SOC := qca9563
+  DEVICE_VENDOR := Qxwlan
+  DEVICE_MODEL := E1700AC V2 (16M flash)
+  DEVICE_PACKAGES := kmod-usb2 kmod-ath10k-ct ath10k-firmware-qca988x-ct -swconfig
+  IMAGE_SIZE := 15936k
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | \
+	append-metadata | check-size
+endef
+TARGET_DEVICES += qxwlan_e1700ac-v2-16m
+
+define Device/qxwlan_e1700ac-v2-8m
+  $(Device/qxwlan_e1700ac-v2-16m)
```

Comment:
Please do not derive on device from another one. Create a shared node instead, and base both devices on it, e.g. Device/dlink_dap-13xx in the same file.